src/components/PDF/ApplicationForm.tsx

```diff
@@ -290,37 +333,57 @@ class ApplicationForm extends Component<Props, State> {
                                 id={entry.fieldName}
```

Comment:
fieldId

Reply:
Well, actually it works with fieldName because fieldID is frontend ID (just to satisfy react complaining about keys), while fieldName is the backend ID that is used to identify the fields on the server